app/src/Client/ClientApi.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace Client;
+
+use Application\BaseApi;
+use User\UserApi;
+
+class ClientApi extends BaseApi
+{
+    /**
+     * Get all clients associated with the current user
+     *
+     * @return array
+     */
+    public function getCollection($queryParams)
+    {
+        $talks_uri = $this->baseApiUrl . '/v2.1/applications';
+
+        $talks = (array) json_decode(
```

Comment:
Doesn't work as the "internal" structure needs to be objects. Casting to an array only casts the outer object to an array while the internal objects still remain objects.

Would be a great idea but doesn't work in this special case without refactoring a large portion of the site. 😄